app/views/homes/new.html.erb

```diff
@@ -0,0 +1 @@
+<div id="app"></div>
```

Comment:
looks like only "homes#index" is being used in `routes.rb`, so we can go ahead and remove this